### ISSUE TITLE
doccano init fails with importerror

### ISSUE THREAD

**Issue by gramhagen** [doccano/doccano#1446]
How to reproduce the behaviour
---------

Your Environment
---------

*   Operating System: ubuntu 20.04 lts
*   Python Version Used: 3.8
*   When you install doccano:
*   How did you install doccano (Heroku button etc): pip install doccano (which used version == 1.4.1)

the following error occurs after running `doccano init`
```
azureuser@doccano-server:~$ doccano init
Setup Database.
Filepond app init: Creating temporary file upload directory </home/azureuser/.local/lib/python3.8/site-packages/backend/filepond-temp-uploads>
Filepond app init: Creating file store directory </home/azureuser/.local/lib/python3.8/site-packages/backend/media>...
Traceback (most recent call last):
  File "/home/azureuser/.local/lib/python3.8/site-packages/backend/manage.py", line 15, in <module>
    execute_from_command_line(sys.argv)
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
    utility.execute()
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/core/management/__init__.py", line 413, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/core/management/base.py", line 354, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/core/management/base.py", line 393, in execute
    self.check()
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/core/management/base.py", line 419, in check
    all_issues = checks.run_checks(
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/core/checks/registry.py", line 76, in run_checks
    new_errors = check(app_configs=app_configs, databases=databases)
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/core/checks/urls.py", line 13, in check_url_config
    return check_resolver(resolver)
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/core/checks/urls.py", line 23, in check_resolver
    return check_method()
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/urls/resolvers.py", line 412, in check
    for pattern in self.url_patterns:
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/utils/functional.py", line 48, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/urls/resolvers.py", line 598, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/utils/functional.py", line 48, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/urls/resolvers.py", line 591, in urlconf_module
    return import_module(self.urlconf_name)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/azureuser/.local/lib/python3.8/site-packages/backend/app/urls.py", line 44, in <module>
    path('v1/', include('api.urls')),
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/urls/conf.py", line 34, in include
    urlconf_module = import_module(urlconf_module)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/azureuser/.local/lib/python3.8/site-packages/backend/api/urls.py", line 165, in <module>
    path('auth/', include('dj_rest_auth.urls')),
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/urls/conf.py", line 34, in include
    urlconf_module = import_module(urlconf_module)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/azureuser/.local/lib/python3.8/site-packages/dj_rest_auth/urls.py", line 4, in <module>
    from dj_rest_auth.views import (
  File "/home/azureuser/.local/lib/python3.8/site-packages/dj_rest_auth/views.py", line 16, in <module>
    from .app_settings import (
  File "/home/azureuser/.local/lib/python3.8/site-packages/dj_rest_auth/app_settings.py", line 3, in <module>
    from dj_rest_auth.serializers import JWTSerializer as DefaultJWTSerializer
  File "/home/azureuser/.local/lib/python3.8/site-packages/dj_rest_auth/serializers.py", line 22, in <module>
    from .forms import PasswordResetForm
  File "/home/azureuser/.local/lib/python3.8/site-packages/dj_rest_auth/forms.py", line 16, in <module>
    from django.contrib.auth.forms import DefaultPasswordResetForm
ImportError: cannot import name 'DefaultPasswordResetForm' from 'django.contrib.auth.forms' (/home/azureuser/.local/lib/python3.8/site-packages/django/contrib/auth/forms.py)
Traceback (most recent call last):
  File "/home/azureuser/.local/lib/python3.8/site-packages/backend/manage.py", line 15, in <module>
    execute_from_command_line(sys.argv)
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
    utility.execute()
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/core/management/__init__.py", line 413, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/core/management/base.py", line 354, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/core/management/base.py", line 398, in execute
    output = self.handle(*args, **options)
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/core/management/base.py", line 89, in wrapped
    res = handle_func(*args, **kwargs)
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/core/management/commands/migrate.py", line 75, in handle
    self.check(databases=[database])
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/core/management/base.py", line 419, in check
    all_issues = checks.run_checks(
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/core/checks/registry.py", line 76, in run_checks
    new_errors = check(app_configs=app_configs, databases=databases)
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/core/checks/urls.py", line 13, in check_url_config
    return check_resolver(resolver)
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/core/checks/urls.py", line 23, in check_resolver
    return check_method()
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/urls/resolvers.py", line 412, in check
    for pattern in self.url_patterns:
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/utils/functional.py", line 48, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/urls/resolvers.py", line 598, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/utils/functional.py", line 48, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/urls/resolvers.py", line 591, in urlconf_module
    return import_module(self.urlconf_name)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/azureuser/.local/lib/python3.8/site-packages/backend/app/urls.py", line 44, in <module>
    path('v1/', include('api.urls')),
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/urls/conf.py", line 34, in include
    urlconf_module = import_module(urlconf_module)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/azureuser/.local/lib/python3.8/site-packages/backend/api/urls.py", line 165, in <module>
    path('auth/', include('dj_rest_auth.urls')),
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/urls/conf.py", line 34, in include
    urlconf_module = import_module(urlconf_module)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/azureuser/.local/lib/python3.8/site-packages/dj_rest_auth/urls.py", line 4, in <module>
    from dj_rest_auth.views import (
  File "/home/azureuser/.local/lib/python3.8/site-packages/dj_rest_auth/views.py", line 16, in <module>
    from .app_settings import (
  File "/home/azureuser/.local/lib/python3.8/site-packages/dj_rest_auth/app_settings.py", line 3, in <module>
    from dj_rest_auth.serializers import JWTSerializer as DefaultJWTSerializer
  File "/home/azureuser/.local/lib/python3.8/site-packages/dj_rest_auth/serializers.py", line 22, in <module>
    from .forms import PasswordResetForm
  File "/home/azureuser/.local/lib/python3.8/site-packages/dj_rest_auth/forms.py", line 16, in <module>
    from django.contrib.auth.forms import DefaultPasswordResetForm
ImportError: cannot import name 'DefaultPasswordResetForm' from 'django.contrib.auth.forms' (/home/azureuser/.local/lib/python3.8/site-packages/django/contrib/auth/forms.py)
Traceback (most recent call last):
  File "/home/azureuser/.local/lib/python3.8/site-packages/backend/manage.py", line 15, in <module>
    execute_from_command_line(sys.argv)
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
    utility.execute()
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/core/management/__init__.py", line 413, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/core/management/base.py", line 354, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/core/management/base.py", line 393, in execute
    self.check()
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/core/management/base.py", line 419, in check
    all_issues = checks.run_checks(
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/core/checks/registry.py", line 76, in run_checks
    new_errors = check(app_configs=app_configs, databases=databases)
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/core/checks/urls.py", line 13, in check_url_config
    return check_resolver(resolver)
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/core/checks/urls.py", line 23, in check_resolver
    return check_method()
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/urls/resolvers.py", line 412, in check
    for pattern in self.url_patterns:
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/utils/functional.py", line 48, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/urls/resolvers.py", line 598, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/utils/functional.py", line 48, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/urls/resolvers.py", line 591, in urlconf_module
    return import_module(self.urlconf_name)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/azureuser/.local/lib/python3.8/site-packages/backend/app/urls.py", line 44, in <module>
    path('v1/', include('api.urls')),
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/urls/conf.py", line 34, in include
    urlconf_module = import_module(urlconf_module)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/azureuser/.local/lib/python3.8/site-packages/backend/api/urls.py", line 165, in <module>
    path('auth/', include('dj_rest_auth.urls')),
  File "/home/azureuser/.local/lib/python3.8/site-packages/django/urls/conf.py", line 34, in include
    urlconf_module = import_module(urlconf_module)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/azureuser/.local/lib/python3.8/site-packages/dj_rest_auth/urls.py", line 4, in <module>
    from dj_rest_auth.views import (
  File "/home/azureuser/.local/lib/python3.8/site-packages/dj_rest_auth/views.py", line 16, in <module>
    from .app_settings import (
  File "/home/azureuser/.local/lib/python3.8/site-packages/dj_rest_auth/app_settings.py", line 3, in <module>
    from dj_rest_auth.serializers import JWTSerializer as DefaultJWTSerializer
  File "/home/azureuser/.local/lib/python3.8/site-packages/dj_rest_auth/serializers.py", line 22, in <module>
    from .forms import PasswordResetForm
  File "/home/azureuser/.local/lib/python3.8/site-packages/dj_rest_auth/forms.py", line 16, in <module>
    from django.contrib.auth.forms import DefaultPasswordResetForm
ImportError: cannot import name 'DefaultPasswordResetForm' from 'django.contrib.auth.forms' (/home/azureuser/.local/lib/python3.8/site-packages/django/contrib/auth/forms.py)
```

I believe this is because of breaking changes in dj-rest-auth package. I downgraded that from 2.1.8 -> 2.1.5 and the issue went away.

**Comment by afiqmuzaffar** [comment]
Thanks for opening up this issue, it's working!

**Comment by Hironsan** [comment]
This problem is fixed in [dj-rest-auth>=2.1.9](https://github.com/iMerica/dj-rest-auth/releases/tag/2.1.9):

- https://github.com/iMerica/dj-rest-auth/issues/295
